demisto_sdk/commands/common/update_id_set.py

```diff
@@ -135,6 +135,29 @@ def does_dict_have_alternative_key(data: dict) -> bool:
     return False
 
 
+def get_alternative_fields_from_data(data: dict) -> Union[bool, dict]:
```

Comment:
But this function should return also whether there is any alternative key in the file (that is not id/name). this modifies the `has_alternative_meta` field in the id set.
if we return an empty dict, this will not work.
if we want to return the exact fields (that are not the id/name) then this means we need to change the recursive function `does_dict_have_alternative_key`, and iterate over the nested dicts in a playbook and save the values to a new dict, and this could be a lot of work for data we don't actually need. 
WDYT?